### ISSUE TITLE
files in snippets needs clean up

### ISSUE THREAD

**Issue by hrani** [BhallaLab/moose-examples#13]
Following scripts do not run due to deprecated API

- [x] Izhikevich_with_synapse.py  Commenting these line 88 to 92 and103, runs the model
- [x] testSigNeur.py 
- [x] wildcard.py

runs but gives

```
Elements Matching:', '/alfa/##[FIELD(Gbar)=1e-6]')
Error: SetGet::strGet: Field Gbar not found on Element gateX
```

**Comment by subhacom** [comment]
> 3.wildcard.py
> runs but gives
> Elements Matching:', '/alfa/##[FIELD(Gbar)=1e-6]')
> Error: SetGet::strGet: Field Gbar not found on Element gateX
>
> 

I suspect this is expected (and known), because there are elements like
HHGate under `/alfa` which do not have the field `Gbar`. A better
alternative would be for SetGet to not print error message for this, at
least in the non-debug build. It could return an error code instead which
can be handled by higher level functions.

**Comment by dilawar** [comment]
@subhacom Would you suggest c++ to throw exception (equivalent of Python KeyError/IndexException) which is to be handled by `pymoose`?

**Comment by subhacom** [comment]
If I remember correctly SetGet::strSet and SetGet::strGet return bool to indicate success or failure. This is inspected in pymoose and Exception is thrown as suitable. However, when these are used in internal C++ code of MOOSE (in this case wildcardFind), the function should handle it locally. These functions are so fundamental that it does not make sense to print error messages for this. 
Also, I am not sure about the performance cost of throwing exception at the base code level.

**Comment by dilawar** [comment]
Only testSigNeur.py is failing now with the following error.
```
AttributeError: 'moose.SynChan' object has no attribute 'synapse'
```

**Comment by dilawar** [comment]
This ticked is closed in the favor of #30
